tests/network/primary_network/test_attach_network.py

```
@@ -0,0 +1,129 @@
+    from kubernetes.dynamic import DynamicClient
+    from ocp_resources.namespace import Namespace
+
+    from libs.vm.vm import BaseVirtualMachine
+
+LOGGER = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def vm_without_pod_interface(
+    namespace: Namespace, unprivileged_client: DynamicClient
+) -> Generator[BaseVirtualMachine, None, None]:
```

Ruff (0.15.14)

[warning] Unnecessary default type arguments

Remove default type arguments

(UP043)